google/cloud/internal/parse_rfc3339.cc

```diff
@@ -14,218 +14,25 @@
 
 #include "google/cloud/internal/parse_rfc3339.h"
 #include "google/cloud/internal/throw_delegate.h"
-#include <array>
-#include <cctype>
-#include <cstdio>
-#include <iomanip>
-#include <iostream>
-#include <sstream>
-
-namespace {
-[[noreturn]] void ReportError(std::string const& timestamp, char const* msg) {
-  std::ostringstream os;
-  os << "Error parsing RFC 3339 timestamp: " << msg
-     << " Valid format is YYYY-MM-DD[Tt]HH:MM:SS[.s+](Z|[+-]HH:MM), got="
-     << timestamp;
-  google::cloud::internal::ThrowInvalidArgument(os.str());
-}
```

Comment:
The version of this PR that I've had laying around kept (a slightly tweaked) `ReportError()`-like message because the error string from `absl::ParseTime()` is notoriously terse.

Reply:
Good point. Fixed.